docs/getting-started.md

```diff
@@ -5,7 +5,19 @@ sidebar_label: Getting started
 
 # Getting Started with Divvi Mobile
 
-Creating a mobile app with Divvi Mobile is incredibly quick and straightforward. In just a few steps, you'll have your own mobile app ready for development!
+Build a fully functional blockchain wallet app in minutes with just a few lines of code. Divvi Mobile provides a complete foundation, including wallet setup, token transactions, and onboarding, so that you can focus on what makes your app unique.
+
+## Why Use Divvi Mobile?
+
+✅ **Launch faster** – Spin up a mobile wallet app with built-in onboarding, cash-in, swap, and send flows.
+
+✅ **Customizable look and feel** – Adjust feature visibility, colors, and assets to match your brand.
+
+✅ **Flexible integrations** – Add custom screens and flows while keeping essential wallet functionality intact.
+
+✅ **Divvi Protocol built in** – Earn rewards from the value your users bring to integrated protocols.
+
+With Divvi Mobile, you don’t have to worry about how users will acquire a wallet or tokens. Instead, you can focus on what sets your app apart while Divvi Mobile ensures your users have everything they need to get started.
```

Comment:
This is good, though I think it should go in the overview. So the getting started is short and actionable.
Assuming people who look at this guide already know why that want to get started.

I guess we probably need to take a broak look at this between the repo README, Architecture and the overview.

Reply:
yes i was wondering the same, i can move this to the overview.